cmd/commands/runtime.go

```
@@ -878,6 +880,14 @@ func runRuntimeUpgrade(ctx context.Context, opts *RuntimeUpgradeOptions) error {
 		return err
 	}
 
+	requiredCliVersionConstraint, _ := semver.NewConstraint(newRt.Spec.RequiredCLIVersion)
```

explain why ignoring the err here, what are the possible flows here